### PR TITLE
chore: add default templates to create new issues and raise pull requests

### PR DESCRIPTION
**Pull request by apsinghdev**
fixes: #4102

This PR adds default templates to create new issues and raise pull requests.

Screenshot 2024-10-09 at 12 04 57 AM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### Screenshots
+
+<!-- Add screenshots, if applicable, to help explain your problem. -->
+
+### Environment:
+
+-   Operating System: [e.g., Windows, macOS, Linux]
+-   Browser (if applicable): [e.g., Chrome, Firefox, Safari]
+
+### Checklist
```
Comment (frankie567) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I would remove that checklist, not so useful in my experience 😄

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+-   [ ] I have read and followed the project's code of conduct.
+-   [ ] I have searched for similar issues before creating this one.
+-   [ ] I have provided steps to reproduce the issue.
+-   [ ] I have provided all the necessary information to understand and reproduce the issue.
+-   [ ] I am willing to contribute to the resolution of this issue if approved.
+
+---
```
Comment (frankie567) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: Love that last part (lines 38-42) 😄 However, I would put it in HTML comments, it doesn't have to appear in the body of the issue.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: 💡 Feature Request
+    url: https://github.com/orgs/polarsource/discussions/new?category=feature-requests
+    about: Please submit feature requests here.
+  - name: 🛟 Improvement Request
```
Comment (frankie567) [inline .github/ISSUE_TEMPLATE/config.yml]: This is a bit redundant with Feature Request. I would prefer to have a support link to our Discord. Something like:

```yml
  - name: 🛟 Support
    url: https://discord.com/invite/STfRufb32V
    about: Join us on Discord to ask support questions
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+## Description
```
Comment (frankie567) [inline .github/PULL_REQUEST_TEMPLATE.md]: I appreciate your effort of setting up a PR template; however, I would remove it.

Currently, most of our PR are either done by Polar team or by contributors we already gave guidance to on Discord and/or GitHub issues. So, I don't think we need it right now :)

**Review by frankie567 — CHANGES_REQUESTED** [review]
Thank you for this, @apsinghdev! That's really helpful and will clearly help newcomers being guided to the right channel.

Just a few tweaks and we should be good to go 👍

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+name: 🪲 Bug Report
+about: Report an issue to help us improve
+title: ""
+labels: "Issue-Bug"
```
Comment (frankie567) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Our label is `bug`

**Comment by apsinghdev** [comment]
@frankie567 Thanks a lot for the feedback! I have updated the PR with the necessary changes. Please have a look.

**Comment by frankie567** [comment]
Amazing! Thank you for this @apsinghdev 🙏

**Comment by apsinghdev** [comment]
> Amazing! Thank you for this @apsinghdev 🙏

Thanks!

**Pull request merged**